cmd/expiration-mailer/main.go

```diff
@@ -399,9 +398,8 @@ type Config struct {
 
 		CertLimit int
 		NagTimes  []string
-		// How much earlier (than configured nag intervals) to
-		// send reminders, to account for the expected delay
-		// before the next expiration-mailer invocation.
+
+		// Deprecated: https://github.com/letsencrypt/boulder/issues/6097
```

Comment:
I personally prefer to also have a `TODO(#6097): Remove this` so that it is super easy to find in the future, but up to you.